dev/vhost.conf

```diff
@@ -0,0 +1,20 @@
+<VirtualHost *:80>
```

Comment:
There is already vhost in the server/vhost.conf as the default configuration. I think it's better not to force the domain name by setting server name. Service "server" won't serve multiple sites that needs that.

Reply:
ServerName is insignificant here anyway. I've added vhost.conf file just because there was a directory named vhost.conf and I thought you needed the file, but created a directory by mistake. Now I see that there's no dev directory on master branch.. no idea what happened here, but after cloning master branch it came with dev directory.